chatterbox/src/Main/Header/index.js

```diff
@@ -0,0 +1,132 @@
+                                <Button raised className={classes.button}>
+                                    New Chat
+                                </Button>
+                                &nbsp;&nbsp;
+                                <Button raised className={classes.button}>
+                                    New Group
+                                </Button>
+                            </div>
+                        </div>
+
+                        {auth && (
+                            <div className="appbar-buttons-right">
+                                <Button
+                                    aria-owns={open ? 'menu-appbar' : null}
+                                    aria-haspopup="true"
+                                    onClick={this.handleMenu}
+                                >
+                                    <h6 id="Addie_div">Addie Hogan </h6> &nbsp;
+                                    <img
+                                        className={classes.images}
+                                        height="60"
+                                        src={Oval}
+                                        alt=" "
+                                    />
+                                </Button>
+
+                                <Button aria-owns={open ? 'menu-appbar' : null} color="contrast">
+                                    <img
+                                        src={Oval_2}
+                                        height="60"
+                                        alt=" "
+                                        className={classes.images}
+                                    />
+                                </Button>
+                                <Menu
+                                    id="menu-appbar"
+                                    anchorEl={anchorEl}
+                                    anchorOrigin={{
+                                        vertical: 'top',
+                                        horizontal: 'center'
+                                    }}
+                                    transformOrigin={{
+                                        vertical: 'top',
+                                        horizontal: 'center',
+                                        width: '50'
+                                    }}
+                                    open={open}
+                                    onClose={this.handleClose}
+                                >
+                                    <MenuItem onClick={this.handleClose}>Settings</MenuItem>{' '}
+                                    {/*redirect to seetings */}
+                                    <MenuItem onClick={this.handleClose}>Log Out</MenuItem>
+                                </Menu>
+                            </div>
+                        )}
+                    </Toolbar>
+                </AppBar>
+            </div>
+        );
+    }
+}
+
+MenuAppBar.propTypes = {
```

Comment:
I think you can move the propTypes to the MenuAppBar class and just make it static.

Reply:
Moved propTypes to the MenuAppBar class and made it to be static